hugo/plan.sh

```diff
@@ -1,15 +1,15 @@
 pkg_name=hugo
 pkg_origin=core
-pkg_version=0.56.3
+pkg_version=0.57.0
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=("Apache-2.0")
 pkg_description="Hugo is one of the most popular open-source static site generators."
 pkg_build_deps=(
   core/go
   core/git
-  core/gcc
 )
 pkg_deps=(
+  core/gcc
```

Comment:
Is there a reason we can't use gcc-libs here?  It should provide the necessary `ibstdc++` without shipping a whole compiler as a dep :smile: